datahub-web-react/src/app/entity/shared/containers/profile/header/PlatformContent/PlatformContentView.tsx

```diff
@@ -152,6 +159,16 @@ function PlatformContentView(props: Props) {
                 </ParentContainersWrapper>
                 {directParentContainer && <ContainerLink container={directParentContainer} />}
             </StyledTooltip>
+            {parentDatasetUrn && parentDatasetProperties && parentDatasetIcon && (
+                <span>
+                    <StyledRightOutlined data-testid="right-arrow" />
+                    <DatasetLink
+                        parentDatasetUrn={parentDatasetUrn}
+                        parentDatasetProperties={parentDatasetProperties}
+                        parentDatasetIcon={parentDatasetIcon}
+                    />
+                </span>
+            )}
```

Comment:
would you mind posting a screenshot of the final output in your PR description so we can see how this change looks in the UI?

Reply:
<img width="774" alt="Screenshot 2024-10-27 at 10 26 13 PM" src="https://github.com/user-attachments/assets/742afa84-c10c-44e2-abc3-633c7fd75838">